src/ocha_stratus/datasources/worldpop.py

```diff
@@ -0,0 +1,105 @@
+import urllib.request
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+import xarray as xr
+from pystac_client import Client
+
+WORLDPOP_STAC_URL = "https://api.stac.worldpop.org"
```

Comment:
Related to my comment (https://github.com/OCHA-DAP/ocha-stratus/pull/29#issuecomment-3576713958), why don't we start building the "useful stac endpoint catalogue" and just have one entry in it for now ("worldpop") and then we can load the `WORLDPOP_STAC_URL` like this. This could be a kind of happy medium between building an immediately useful tool and working towards something more flexible.

```
stratus.stac_endpoint("worldpop")
```